generator/src/OperatorTestGenerator.php

```diff
@@ -158,6 +162,7 @@ private function convertYamlTaggedValues(mixed $object): mixed
         }
 
         if (is_object($object)) {
+            $object = clone $object;
```

Comment:
This is kind of but that doesn't affect the current code. When calling `$pipeline = $this->convertYamlTaggedValues($test->pipeline);`, the original `$test->pipeline` is modified, which is not expected.

I hit this issue because I hacked into this file to generate the pipeline of test methods with some dumb implementation. I did not commit that because that was not good.

Reply:
Oh, I originally missed that this was in the generator sources. Glad it's fixed and I see no need for a PHPLIB ticket to document this since the code is internal.